packages/shared/src/components/cards/v1/ActionButtons.tsx

```diff
@@ -102,26 +103,23 @@ export default function ActionButtons({
             }
           >
             <Button
-              className="pointer-events-auto hover:text-theme-color-avocado"
+              className={classNames(
+                'pointer-events-auto',
+                post?.numUpvotes > 0 ? '!pl-1 !pr-3' : '!px-1',
```

Comment:
I think once this experiment is concluded we should definitely look at some refactoring of these buttons and custom stuff to be more aligned with our styling guidelines.

Reply:
Yes, we should then probably create a proper `ActionButton` implementation. This is just a hack.

Ideally, we would be able to set custom padding and icon size on the `ButtonV2` component. Didn't know it yet when I was refactoring the old one 😭

Reply:
@nensidosari Let's discuss this with Tsahi today if we can get a clear updated guidelines somewhere as I get a bit lost a button types tbh.

Reply:
Same here